src/core_plugins/timelion/server/handlers/__tests__/parse_sheet.js

```diff
@@ -0,0 +1,50 @@
+const chainRunnerFn = require('../chain_runner');
+const tlConfig = require('../../series_functions/__tests__/fixtures/tlConfig.js')();
+
+const expect = require('chai').expect;
+
+describe('timelion parse_sheet function', function () {
+  const chainRunner =  chainRunnerFn(tlConfig);
+
+  async function parse(data) {
+    const request = {
+      sheet: data,
+      time: {
+        from: 'now-15m',
+        to: 'now',
+        mode: 'quick',
+        interval: 'auto',
+        timezone: 'Europe/Berlin'
+      }
+    };
+    return await Promise.all(chainRunner.processRequest(request));
```

Comment:
this does not work  .... it throws `object promise is not iterable` however the same works in `routes/run.js`
no idea whats wrong ... for some reason it doesn't wait here for the promises to resolve no matter what i try. help appreciated!